lib/DDG/Spice/Pubmed/Search.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Pubmed::Search;
```

Comment:
That makes sense about changing the alt_to and ajax calls. I'll leave as is, but thanks for taking the time to explain it out.